cli/packages/visualize/visualize.go

```diff
@@ -16,8 +22,35 @@ type TableOptions struct {
 // 	}
 // }
 
+const (
+	// combined width of the table borders and padding
+	borderWidths = 10
+	// char to indicate that a string has been truncated
+	ellipsis = "…"
+)
+
 // Given headers and rows, this function will print out a table
-func Table(headers []string, rows [][]string) {
+func Table(headers [3]string, rows [][3]string) {
```

Comment:
This method was intended to be used for any number of rows and colums. Is there a reason why you are restricting it to 3? I understand that 3 is for the case of displaying secrets, but i feel like we lose the ability to use it for any number of rows/cols

Reply:
I made the 3 rows explicit because adding new rows will break the rendering. 
Since the terminal has only limited width and there's no horizontal scrolling, you have to calculate available space yourself and explicitly define where to truncate values. 
A generic table will almost never work unless you'd specify rules like "always truncate the first column" for example. But that doesn't make much sense imo.